examples/js/workers/OffscreenCanvas.js

```diff
@@ -52,7 +58,10 @@ function animate() {
 	mesh.rotation.y += delta * 0.5;
 
 	renderer.render( scene, camera );
-
-	renderer.context.commit().then( animate );
+	if (self.requestAnimationFrame) {
```

Comment:
Please respect the code style. E.g. instead of your code do this:
```
if ( self.requestAnimationFrame ) {

    self.requestAnimationFrame( animate );

} else ( renderer.context.commit ) {

    renderer.context.commit().then( animate );

}
```